molecule/default/molecule.yml

```diff
@@ -10,8 +11,13 @@ platforms:
 provisioner:
   name: ansible
   log: true
+  playbooks:
+    prepare: ../common/prepare.yml
 verifier:
   name: ansible
+lint: |
```

Comment:
Oh maybe I should try this too instead of the extra independent call to ansible-lint.

Reply:
IIRC, your call to ansible-lint is just covering tests/test.yml; which I don't think lints the rest of the repo.